# save the current joint states of the arm under the given name
string name
---
string result
